packages/vite/src/node/plugins/modulePreloadPolyfill.ts

```diff
@@ -78,13 +78,13 @@ function polyfill() {
     }
   }).observe(document, { childList: true, subtree: true })
 
-  function getFetchOpts(script: any) {
+  function getFetchOpts(link: any) {
     const fetchOpts = {} as any
-    if (script.integrity) fetchOpts.integrity = script.integrity
-    if (script.referrerpolicy) fetchOpts.referrerPolicy = script.referrerpolicy
-    if (script.crossorigin === 'use-credentials')
+    if (link.integrity) fetchOpts.integrity = link.integrity
+    if (link.referrerpolicy) fetchOpts.referrerPolicy = link.referrerpolicy
```

Comment:
```suggestion
    if (link.referrerPolicy) fetchOpts.referrerPolicy = link.referrerPolicy
```

```js
'referrerPolicy' in document.createElement('link') // true
'referrerpolicy' in document.createElement('link') // false
```
It seems this one is also wrong.